test/manual-test-examples/webgl/pixels/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title></title>
+  <link rel="stylesheet" href="">
+  <script language="javascript" type="text/javascript" src="../../../../lib/p5.js"></script>
+  <script language="javascript" type="text/javascript" src="sketch.js"></script>
+  <style>
+    html, body {margin:0; padding:0;}
+  </style> 
+</head>
+<body>
+<h2 style='margin-left:100px'>If you click the canvas, the box should get textured with an image of the canvas</h2>
```

Comment:
Okay after more investigation. I tried several different approaches and this behavior seems to be the same across this branch and master in all scenarios. I got pixels from a 2d canvas and made them the texture in master and got the same barely present effect. This seems specific to pointLight and has something to do with how it is interacting with a p5.Image returned from get(). I will keep investigating but seems unrelated to this PR. I took the pointLight out of the example so that expected output is the same across browsers.